syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package exam_proto;

message BRD_ctrl{
  required fixed32 command = 1;
  required fixed32 out = 2;
  required sfixed32 status = 3;
}

